templates/page--user--login.html.twig

```diff
@@ -0,0 +1,13 @@
+{%- include "@saa_victoria_subtheme/components/brand-bar/saa--brand-bar.html.twig" with { modifier_class : "su-brand-bar--dark" } -%}
```

Comment:
I'm using this theme file, but you have more text in your preview. Am I missing a commit for this template?